src/components/App.tsx

```diff
@@ -28,11 +29,14 @@ const typeRegistry = new Registry([
 
 export function App(): JSX.Element {
   const [nodeUrl, setNodeUrl] = React.useState(nodeUrls[0]);
+  const [signingClient, setSigningClient] = React.useState<SigningCosmWasmClient>();
```

Comment:
Do we need this state here in `App`? Can't we just have everthing in the context?